docs/Product-Guides/Benefits-API.md

```diff
@@ -1,40 +1,253 @@
+**403(b)** | ✓ | ✓ | ✓ | ✓ |
+**Roth 403(b)** | ✓ | ✓ | ✓ | ✓ |
+**457** | n/a | n/a | n/a | n/a |
+**Roth 457**  | n/a | n/a | n/a | n/a |
+**Simple IRA** | ✓ | ✓ | ✓ | ✓ |
+**HSA (post-tax)** | n/a | n/a | n/a | n/a |
+**HSA (pre-tax)** | ✓ | ✓ | ✓ | ✓ |
+**FSA Dependent** | ✓ | ✓ | ✓ | ✓ |
+**FSA Medical** | ✓ | ✓ | ✓ | ✓ |
+**Section 125 Dental** | ✓ | ✓ | ✓ | ✓ |
+**Section 125 Medical** | ✓ | ✓ | ✓ | ✓ |
+**Section 125 Vision** | ✓ | ✓ | ✓ | ✓ |
+**Commuter (pre-tax)** | ✓ | ✓ | ✓ | ✓ |
+**Custom pre-tax** | n/a | n/a | n/a | n/a |
+**Custom post-tax** | ✓ | ✓ | ✓ | ✓ |
+
+<!--
+type: tab
+title: Supported Features
+-->
+
+Benefit | Features
+--------|----------
+**401(k)** | `company_contribution`: `fixed`, `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`<br>`annual_maximum`
+**Roth 401(k)** | `company_contribution`: `fixed`, `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`<br>`annual_maximum`
+**403(b)** | `company_contribution`: `fixed`, `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`<br>`annual_maximum`
+**Roth 403(b)**| `company_contribution`: `fixed`, `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`<br>`annual_maximum`
+**457**| `company_contribution`: `fixed`, `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`<br>`annual_maximum`
+**Roth 457** | `company_contribution`: `fixed`, `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`<br>`annual_maximum`
+**Simple IRA**| `company_contribution`: `fixed`, `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`<br>`annual_maximum`
+**HSA (pre-tax)**| `company_contribution`: `fixed`, `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`<br>`annual_maximum`<br>`hsa_contribution_limit`: `individual`, `family`
+**FSA Dependent** | `company_contribution`: `fixed`<br>`employee_deduction`: `fixed`
+**FSA Medical** | `company_contribution`: `fixed`<br>`employee_deduction`: `fixed`
+**Section 125 Dental** | `company_contribution`: `fixed`<br>`employee_deduction`: `fixed`
+**Section 125 Medical**| `company_contribution`: `fixed`<br>`employee_deduction`: `fixed`
+**Section 125 Vision** | `company_contribution`: `fixed`<br>`employee_deduction`: `fixed`
+**Commuter (pre-tax)** | `company_contribution`: `fixed` <br>`employee_deduction`: `fixed`
+**Custom post-tax** | `employee_deduction`: `fixed`, `percent`<br>`annual_maximum`
 
-### Enrolling individuals into a benefit
-Enroll individuals into a benefit using [`POST /benefits/{benefit_id}/individuals`](https://developer.tryfinch.com/docs/reference/b3A6MjE4NDU0NTU-enroll-individuals-in-benefits). The request allows you to uniquely configure each individuals enrollment into a benefit.
+<!-- type: tab-end -->
 
-The schema of the configuration varies depending on the type of benefit you are enrolling, so please refer to the docs to ensure you are using the right schema.
+## Getting access to the test Gusto instance
```

Comment:
Would prefer leaving this section out completely from our public docs and providing information as needed.